lib/generator.js

```diff
@@ -98,7 +98,7 @@
         var self = this;
         
         self._options = options || {};
-        self._config = options.config || {};
+        self._config = self._options.config || {};
         
         _logger.debug("Launching with config:\n%s", JSON.stringify(self._config, null, "  "));
         if (options.photoshopVersion) {
```

Comment:
Yeah actually probably a smarter thing to do for sure. I was gonna drop it in but chose to just drop in `self._options` didn't notice there were other references to `options`.